volume_project/extract/2022/rescu_sqls/rescu_centreline.sql

```diff
@@ -0,0 +1,161 @@
+-- Make a table that links RESCU detectors with the centreline from 2022-07-05
+CREATE TABLE scannon.rescu_cent_20220705 AS (
```

Comment:
There are 14 matches where the direction from detector doesn't match with the centreline name. I think it could be sometimes one 1 centreline represents both direction..... do you mind checking them to make sure thats the case?

```
select direction, detector_id, gen_loc, lf_name, right(lf_name, 1), left(direction, 1)
from scannon.rescu_cent_20220705 a
inner join gis.centreline_20220705 on centreline_id = geo_id
where right(lf_name, 1)!= left(direction, 1)
```

Also there is one gen_loc named `and` 😆 , detector_id `DW0151DEG`

Reply:
Yes I will certainly double check!!!

And also, the east bound lanes of Lakeshore Blvd West and the west bound lanes of Lakeshore Blvd East caused me some :meow_headache:

Yes that `and` should be taken care of via General Improvement Idea 1...

Reply:
K I have checked them all and the locations are correct! And they were all on Lake Shore...

Here's how I checked:
- Looked at the rescu geom
- Looked at the corresponding segment
- Looked at the centreline for `lf_name LIKE '%Lake Shore%'` so that I could see if Lake Shore was represented with 1 or 2 lines
    - If Lake Shore was represented with 2 lines, I made sure Eastbound detectors were mapped to the southern segment and that Westbound detectors were mapped to the northern segment